<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsCategorizedSymbolRendererV2Widget</class>
 <widget class="QWidget" name="QgsCategorizedSymbolRendererV2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>298</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboCategorizedColumn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Symbol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChangeCategorizedSymbol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Color ramp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboCategorizedColorRamp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="viewCategories">
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnAddCategories">
       <property name="text">
        <string>Classify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddCategory">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteCategory">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteAllCategories">
       <property name="text">
        <string>Delete all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnJoinCategories">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Join</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdvanced">
       <property name="text">
        <string>Advanced</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboCategorizedColumn</tabstop>
  <tabstop>btnChangeCategorizedSymbol</tabstop>
  <tabstop>cboCategorizedColorRamp</tabstop>
  <tabstop>viewCategories</tabstop>
  <tabstop>btnAddCategories</tabstop>
  <tabstop>btnDeleteCategory</tabstop>
  <tabstop>btnDeleteAllCategories</tabstop>
  <tabstop>btnJoinCategories</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
